src/hooks/useRequest.js

```diff
@@ -7,13 +7,17 @@ import Constants from '../util/Constants';
 const FIVE_MINUTES_IN_MILLISECONDS = 1000 * 60 * 5;
 
 function useOperation(reactQueryKey, operation, urlKey, dataParam, config) {
-  const { data, status } = useQuery(reactQueryKey, async () => {
-    const url = UrlProvider.getUrl(urlKey);
-    const res = await Request[operation](url, dataParam, config);
-    return res.data;
-  }, {
-    staleTime: FIVE_MINUTES_IN_MILLISECONDS
-  });
+  const { data, status } = useQuery(
+    reactQueryKey,
+    async () => {
+      const url = UrlProvider.getUrl(urlKey);
+      const res = await Request[operation](url, dataParam, config);
+      return res.data;
+    },
+    {
+      staleTime: FIVE_MINUTES_IN_MILLISECONDS,
+    },
+  );
```

Comment:
iam not sure if i liked this indentation.
is there a way to fix it?

Reply:
I don't think there is. The problem here is the third parameter. Prettier format it in a way that each line has it's own parameter (since it could not fit them all in a single line of 120 characters).

Without that parameter it would look more "normal":
```js
  const { data, status } = useQuery(reactQueryKey, async () => {
    const url = UrlProvider.getUrl(urlKey);
    const res = await Request[operation](url, dataParam, config);
    return res.data;
  });
```

Reply:
yeah, the 120characters is bad
did you take a look in the weconnect project?
it gets bizarr when the interface is too big.
![Screen Shot 2020-09-11 at 12 17 17](https://user-images.githubusercontent.com/8363610/92943112-bde51e80-f428-11ea-995b-ec85de96ab2e.png)

Reply:
I'm so used with Prettier that I look at this and I don't think is bizarre 😛